python/ray/rllib/agents/impala/vtrace_policy_graph.py

```diff
@@ -206,8 +208,24 @@ def to_batches(tensor):
             ("prev_actions", prev_actions),
             ("prev_rewards", prev_rewards),
         ]
+
+        # Initialize RND network and loss
+        if self.config["rnd"]:
+            self.rnd_norm_obs = tf.placeholder(tf.float32, [None] + list(observation_space.shape))
+            loss_in.append(("norm_obs", self.rnd_norm_obs))
+
+            self.rnd = RND(self.rnd_norm_obs)
+            self.rnd_loss = self.rnd.loss
+            self.loss.total_loss += self.rnd_loss
+            self.rnd_bonus = self.rnd.intr_rew
+
+            self.rnd_bonus_filter = get_filter("MeanStdFilter", (1,))
+            self.rnd_obs_filter = get_filter("MeanStdFilter", observation_space.shape)
```

Comment:
This can probably be self-contained right? like RND can expose some interface such that most of these don't need to be attributes of the policy graph.

Reply:
like 
```
self.rnd = RND(observation_space)
self.loss.total_loss += self.rnd.loss
```

and below, in postprocess trajectory, do
```
if self.rnd:
   sample_batch["rewards"] += self.rnd.calculate_bonus(sample_batch["obs"])
```